subworkflows/nf-core/initialise/main.nf

```diff
@@ -9,7 +9,7 @@ workflow INITIALISE {
     take:
         version         // bool
         help            // bool
-        validate_params // bool
+        validateParams  // bool
```

Comment:
why `validateParams` and not `validate_params`?

Reply:
I'd rather we keep what we take to reflect the params on the other side as much as possible

Reply:
these are comming from plugins

Reply:
This subworkflow is not a plugin, and for me, is you have
`INITIALISE(params.help, params.validate_params...)`
we should reflect these in the input we `take,` so `help` and `validate_params`

Reply:
All params that come from plugins follow Nextlfow's logic, so they are `camelCase`, all params coming from nf-core are `snake_case`